### PR TITLE
feat: error and success pages for contact page

### PR DESCRIPTION
**Pull request by mortennordseth**
Added a generic error and success page that users can be redirected to on error/success when submitting a form.

### CODE REVIEW

File: src/translations/pages/contact.ts [hunk 1]
```diff
@@ -974,4 +974,38 @@ export const Contact = {
       },
     },
   },
+  success: {
+    backButton: _(
```
Comment (jonasbrunvoll) [inline src/translations/pages/contact.ts:978]: Does  back to the ( 'hovedsiden' 'main page', 'hovedsida') sound better?

File: src/translations/pages/contact.ts [hunk 2]
```diff
@@ -974,4 +974,38 @@ export const Contact = {
...
+    ),
+    info: _(
+      'Vi har mottatt din henvendelse og vil behandle den så raskt som mulig.',
+      'We have received your inquiry and will process it as soon as possible.',
+      'Vi har motteke din førespurnad og vil behandle den så raskt som mogleg.',
+    ),
+  },
+  error: {
+    backButton: _(
+      'Tilbake til skjemaside',
```
Comment (jonasbrunvoll) [inline src/translations/pages/contact.ts:996]: skjemasiden*

File: src/translations/pages/contact.ts (hunk 1 above)
Comment (mortennordseth) [reply src/translations/pages/contact.ts:978]: Not sure, but updated

File: src/translations/pages/contact.ts (hunk 2 above)
Comment (mortennordseth) [reply src/translations/pages/contact.ts]: ✅

**Review by jonasbrunvoll — APPROVED** [review]

**Pull request merged**